checkpoint/handler.go

```diff
@@ -44,7 +44,7 @@ func handleMsgCheckpointAdjust(ctx sdk.Context, msg types.MsgCheckpointAdjust, k
 
 	checkpointObj, err := k.GetCheckpointByNumber(ctx, msg.HeaderIndex)
 	if err != nil {
-		logger.Error("Unable to get checkpoint from db", "error", err)
+		logger.Error("Unable to get checkpoint from db", "error", err, "Header Index-", msg.HeaderIndex)
```

Comment:
Refactor this to:
`logger.Error("Unable to get checkpoint from db", "header index", msg.HeaderIndex, "error", err)`